src/Adapter/Http/Repository/HttpPodRepository.php

```diff
@@ -195,12 +203,12 @@ public function attach(Pod $pod, callable $callable)
     }
 
     /**
-     * @param Pod      $pod
-     * @param callable $callable
-     * @param int      $cursor
-     *
-     * @return int
-     */
+       * @param Pod      $pod
```

Comment:
Any reason for this?

Reply:
I blame PHPStorm for it. :) Let me update it quickly.